endpoint-agents/src/main/java/com/thousandeyes/sdk/endpoint/agents/EndpointAgentsApi.java

```diff
@@ -233,38 +237,55 @@ private ApiRequest.ApiRequestBuilder enableEndpointAgentRequestBuilder(UUID agen
+    return new Paginator<>(cursor -> filterEndpointAgents(cursor, agentSearchRequest, max, aid, expand, includeDeleted),
+                          FilterEndpointAgentsResponse::getAgents);
+
+  }
+  /**
+   * Filter endpoint agents
+   * Retrieves a list of endpoint agents within the specified account group that match the specified filters.  If no agents meet the filter criteria, the API returns an empty array. 
    * @param cursor (Optional) Opaque cursor used for pagination. Clients should use &#x60;next&#x60; value from &#x60;_links&#x60; instead of this parameter. (optional)
+   * @param agentSearchRequest The filter options for advanced search filtering for agents. (required)
+   * @param max (Optional) Maximum number of objects to return. (optional)
    * @param aid A unique identifier associated with your account group. You can retrieve your &#x60;AccountGroupId&#x60; from the &#x60;/account-groups&#x60; endpoint. Note that you must be assigned to the target account group. Specifying this parameter without being assigned to the target account group will result in an error response. (optional)
    * @param expand This optional parameter allows you to control the expansion of test resources associated with the agent. By default, no expansion occurs when this query parameter is omitted. To expand the \&quot;clients\&quot; resource, include the query parameter &#x60;?expand&#x3D;clients&#x60;.  For multiple expansions, you have two options:    * Separate the values with commas. For example, &#x60;?expandAgent&#x3D;clients,tasks&#x60;. * Specify the parameter multiple times. For example, &#x60;?expandAgent&#x3D;clients&amp;expandAgent&#x3D;tasks&#x60;.  This parameter offers flexibility for users to customize the expansion of specific resources related to the agent.  (optional
    * @param includeDeleted When requesting entities, set to &#x60;true&#x60; if you want to see deleted entities. (optional)
    * @return FilterEndpointAgentsResponse
    * @throws ApiException if fails to make API call
    */
-  public FilterEndpointAgentsResponse filterEndpointAgents(AgentSearchRequest agentSearchRequest, Integer max, String cursor, String aid, List<ExpandEndpointAgentOptions> expand, Boolean includeDeleted) throws ApiException {
-    ApiResponse<FilterEndpointAgentsResponse> response = filterEndpointAgentsWithHttpInfo(agentSearchRequest, max, cursor, aid, expand, includeDeleted);
+  public FilterEndpointAgentsResponse filterEndpointAgents(String cursor, AgentSearchRequest agentSearchRequest, Integer max, String aid, List<ExpandEndpointAgentOptions> expand, Boolean includeDeleted) throws ApiException {
+    ApiResponse<FilterEndpointAgentsResponse> response = filterEndpointAgentsWithHttpInfo(cursor, agentSearchRequest, max, aid, expand, includeDeleted);
```

Comment:
i'm not seeing this change in the generator, can u point me to it?

Reply:
That was a hack a had before to fix the cursor when it is last parameter, but i fixed it with a lambda, forgot to update the generate api, just did that

Reply:
https://github.com/thousandeyes/thousandeyes-sdk-java/pull/89/commits/1e2a71e0c9460a0cd1f3652273d337dd03091290